src/apps/bridge/sensor_drivers/rbrCodaSensor.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+#include "FreeRTOS.h"
+#include "abstractSensor.h"
+#include "avgSampler.h"
+#include "bm_rbr_data_msg.h"
+#include "sensorController.h"
+
+#include <stdint.h>
+#include <stdlib.h>
+
+#define RBR_CODA_NUM_SAMPLE_MEMBERS 3
+
+typedef struct rbr_coda_aggregations_s {
+  double temp_mean_deg_c;
+  double pressure_mean_ubar;
+  double pressure_stdev_ubar;
```

Comment:
Both these pressures are still in dbar at this time.

Reply:
cool, i'll change them!